storage/complaints_test.go

```diff
@@ -0,0 +1,60 @@
+package storage
+
+import (
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/mailbadger/app/entities"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestComplaints(t *testing.T) {
+	db := openTestDb()
+	defer func() {
+		err := db.Close()
+		if err != nil {
+			logrus.Error(err)
+		}
+	}()
+
+	store := From(db)
+	now := time.Now().UTC()
+
+	// Test insert complaints
+	complaints := []entities.Complaint{
+		{
+			ID:         1,
+			UserID:     1,
+			CampaignID: 1,
+			Recipient:  "jhon@doe.com",
+			UserAgent:  "android",
+			Type:       "bla",
+			FeedbackID: "bla",
+			CreatedAt:  now,
+		},
+		{
+			ID:         2,
+			UserID:     1,
+			CampaignID: 1,
+			Recipient:  "jhon@email.com",
+			UserAgent:  "windows",
+			Type:       "bla",
+			FeedbackID: "bla",
+			CreatedAt:  now,
+		},
+	}
+	// insert complaints 1
+	err := store.CreateComplaint(&complaints[0])
+	assert.Nil(t, err)
+	// insert complaints 2
+	err = store.CreateComplaint(&complaints[1])
+	assert.Nil(t, err)
+
+	// test get total complaints
+	totalComplaints, err := store.GetTotalComplaints(1)
```

Comment:
@Dzalevski we should also test for 0 total complaints, before inserting. This should be done for all tests from this PR.